### ISSUE TITLE
Cultural insensitivity

### ISSUE THREAD

**Issue by tom-a-horrocks** [jezen/is-thirteen#495]
The number 13 has biblical implications which may offend some users. Please consider fourteen or twelve instead.

**Comment by yvt** [comment]
Agreed, but please do not use fourteen. The number four is [generally avoided](https://en.wikipedia.org/wiki/Tetraphobia) in some East Asian countries due to its similarity to the sound of the word for *death* (死, si).
